.github/prompts/phases/phase-1/t0.md

```diff
@@ -0,0 +1,78 @@
+# Phase 1 – Task 0: Implement SCBD meetings index utility
+
+## Role & Objective
+You are a TypeScript engineer working in this repository. Replace the existing scaffolding under `utils/indexers/scbd/` with a production-ready, pure utility that fetches the CBD meetings index. The function must be free of side effects except through explicitly injected dependencies so it can be reused by Nitro tasks and unit tests.
+
+## Key Context
+- Phase requirements: `.github/requirements/roadmap.md` (see **Phase 1 – Data Fetch (Indexer Task)** bullets for expected behaviour).
+- Slice overview and data sources: `.github/requirements/index.md` and `.github/prompts/init.md` (API sample request and logging guidance).
+- Current scaffolding to replace: `utils/indexers/scbd/index.ts` (placeholder `runIndexingTask`, `fetchIndexRaw`, etc.).
+- Shared record types: `shared/types/records.ts` defines `IndexRecord` and related typings that downstream steps depend upon.
+
+## Deliverable
+Create (or refactor to) a single module at `utils/indexers/scbd.ts` that exports a strongly typed function to fetch meetings from the SCBD index API.
```

Comment:
create. general function that takes params to return meetings or other index schemas.

https://github.com/scbd/online-reporting-tool/blob/master/services/solr.ts

https://github.com/scbd/online-reporting-tool/blob/master/api/solr-index.ts

This is the exact interface for the index.